src/main/java/com/jfrog/ide/idea/scan/GradleScanner.java

```diff
@@ -34,7 +34,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Map;
+import java.util.*;
```

Comment:
no change in the code, so why bring all of java.util?

Reply:
not there anymore